### ISSUE TITLE
Issues Setting Up

### ISSUE THREAD

**Issue by Cmbarrows** [biostars/biostar-handbook-issues#157]
Hello, 

I hope this is okay but I wanted to provide some feedback in setting up the computer. I am a novice and really struggled despite following the directions. I am using a windows computer (windows 10 - education) with the Ubuntu installation through the microsoft store.  

The first issue I had was a the http error many other users seem to be getting with the SSL issues. Specifically the "HTTP 000 CONNECTION FAILED for url". 

What I tried: all of the resolutions including setting the ssl_verify to "false" and "no" with no success. Turning off my firewall.  Uninstalling and reinstalling. 
What worked: Installing the program by Shining Light Productions "Win64 OpenSSL v3.0.0 Light". 

The second issue I had was installing all the tools. Specifically the error messages 
"Encountered problems while solving: 
-nothing provides requested seqkit
-nothing provides requested ..." - I wont list them all. 

What I tried: the resolutions suggested that the install went into the base configuration and not the bioinfo one, which wasn't the case because I ran the script "conda activate bioinfo" before trying to install. Other solutions suggested instructions for setting up the bash initialization scripts weren't followed as detailed in the second part of the Installation chapter, but again I did all those steps. 
What worked: running the following two scripts. 
conda config --add channels bioconda
conda config --add channels conda-forge

Anyway, these solutions were not easy to find (but super easy to apply). I'm not sure why I had so many issues. Hoping this is either useful to someone or makes its way into the book for those of us who could not get the install working. Adding these to the "how to fix problems" chapter and the solutions that work would be super helpful. Digging through closed issues was challenging because many solutions were not written for a new user like deleting specific files or changing path names.

**Comment by StretchLearnsR** [comment]
Had the same issues as OP and appreciate the solutions with the two conda config --add channels.

**Comment by ialbert** [comment]
There is now a new page with step-by-step installation instructions for troubleshooting the quick setup

https://www.biostarhandbook.com/step-by-step.html

**Comment by originade** [comment]
I want to say that I had this error today. I narrowed it down to step #6 "Update conda" as well as skipping to step #7 "Install mamba" (for troubleshooting purposes). I followed all steps in order and the suggestions in this thread and other posts.

I resolved the issue from this comment by touching the files: https://github.com/conda/conda/issues/9948#issuecomment-909989810
I had to do it twice, one to update conda and the second time to install mamba. After that the rest of the manual step by step installation worked.

**Comment by ialbert** [comment]
thanks for following this up with a detailed explanation,

 the problem looks to be more complex, I will add link to you troubleshooting solution to the book. A small but consistent percent of people do report the mysterious 

HTTP 000 CONNECTION FAILED for URL

error that so far I never understood the reasons for, 

the issue you found shows a more complicated cause

**Comment by originade** [comment]
There are many potential solutions in the thread I linked previously from downgrading anaconda versions (not ideal) or installing a previous version then updating, `chmod 777` the anaconda directory, completely restarting the WSL, or touching the files as I did (probably the easiest solution). People seem to think it's a WSL issue with conda.

If anyone is having this HTTP 000 error, I would check out that thread for potential solutions: https://github.com/conda/conda/issues/9948#issuecomment-909989810

I'm glad you've linked it in the book. Thanks!
